src/systems/filecoin_files/data_transfer/_index.md

```diff
@@ -3,4 +3,92 @@ menuTitle: Data Transfer
 title: "Data Transfer in Filecoin"
 ---
 
+_Data Transfer_ is a system for transferring all or part of a `Piece` across the network when a deal is made.
+
+# Terminology
+
+- **Push Request**: A request to send data to the other party
+- **Pull Request**: A request to have the other party send data
+- **Requestor**: The party that initiates the data transfer request (whether Push or Pull)
+- **Responder**: The party that receives the data transfer request
+- **Data Transfer Voucher**: A wrapper around storage or retrieval data that can identify and validate the transfer request to the other party
+- **Request Validator**: The data transfer module only initiates a transfer when the responder can validate that the request is tied directly to either an existing storage deal or retrieval deal. Validation is not performed by the data transfer module itself. Instead, a request validator inspects the data transfer voucher to determine whether to respond to the request.
+- **Scheduler**:  Once a request is negotiated and validated, actual transfer is managed by a scheduler on both sides. The scheduler is part of the data transfer module but is isolated from the negotiation process. It has access to an underlying verifiable transport protocol and uses it to send data and track progress.
+- **Subscriber**: An external component that monitors progress of a data transfer by subscribing to data transfer events, such as progress or completion.
+- **GraphSync**: The default underlying transfer protocol used by the Scheduler. The full graphsync specification can be found at [https://github.com/ipld/specs/blob/master/block-layer/graphsync/graphsync.md](https://github.com/ipld/specs/blob/master/block-layer/graphsync/graphsync.md)
+
+# Request Phases
+
+There are two basic phases to any data transfer:
+
+1. Negotiation - the requestor and responder agree to the transfer by validating with the data transfer voucher
+2. Transfer - Once both parties have negotiated and agreed upon, the data is actually transferred. The default protocol used to do the transfer is Graphsync
+
+Note that the Negotiation and Transfer stages can occur in seperate round trips,
+or potentially the same round trip, where the requesting party implicitly agrees by sending the request, and the responding party can agree and immediately send or receive data.
+
+# Example Flows
+
+## Push Flow
```

Comment:
not sure but there may be an "alternate push flow" possible/useful too that starts sending along with the Data Transfer request, and provides all the data needed to start accepting. this would shave 1 RTT. (maybe just out of scope for now until it's needed)